app/views/campaign_pages/_link_form.slim

```diff
@@ -0,0 +1,28 @@
+.collection-editor
+  ul.list-group
+    - page.links.each do |link|
+      = render 'link', link: link
+
+  = form_for Link.new, remote: true, html: {class: 'form-element', id: 'new_collection_element'}  do |f|
+
+    = f.hidden_field :campaign_page_id, value: page.id
+
+    .col-md-12
+      .form-group.col-md-3
+        = f.label :url, "URL"
```

Comment:
agreed. We could create a liquid tag which exposes rails' i18t method, with a separate yml file for sumofus.